src/plugins/format.js

```diff
@@ -33,6 +33,19 @@
 			pre: 'Preformatted Text'
 		};
 
+		var bbcodeTags = {
+			h1: 'Heading 1',
+			h2: 'Heading 2',
+			h3: 'Heading 3',
+			h4: 'Heading 4',
+			h5: 'Heading 5',
+			h6: 'Heading 6',
+			address: 'Address',
+			pre: 'Preformatted Text'
+		};
+
+		var tags;
```

Comment:
Add a small comment explaining why this is here. Ppl may forget to check the latter lines to understand.

Reply:
done

Reply:
I meant the `var tags;` code

Reply:
Sorry, don't understand what do you mean...
Explain in comments why tags variable is needed?!

Reply:
Somewhat.
It is explaining what the `tags` variable is used for.

Reply:
updated

Reply:
> with the list **form** bbcodeTags or

Change to:
with the list **from** bbcodeTags or

Reply:
updated